src/System.IO.FileSystem/src/System/IO/EnumerationOptions.cs

```diff
@@ -18,10 +18,10 @@ public class EnumerationOptions
         /// explicitly specify EnumerationOptions.
         /// </summary>
         internal static EnumerationOptions Compatible { get; } = new EnumerationOptions
-            { MatchType = MatchType.Win32, AttributesToSkip = 0, IgnoreInaccessible = false };
+        { MatchType = MatchType.Win32, AttributesToSkip = 0, IgnoreInaccessible = false };
```

Comment:
This file has no changes and can be undone.

Reply:
Ah yes, thanks for noticing. I reverted the changes I did to the static EnumerationOptions, but didn't match the spacing with the original.